### PR TITLE
Allow formatting of zero values

### PR DESCRIPTION
**Pull request by davehancock**
We have various use cases where we want to pass zero valued numbers, but would still like them formatted in a consistent way.

E.g A zero valued fee in comparison: https://transferwise.com/gb/compare/

```
[0 - value] [EUR - currency] [ES - lang / locale]
```

Should ideally render as:

```
0,00 EUR
```

Today it will return:

```
null EUR
```

### CODE REVIEW

File: src/number/number.spec.js [hunk 1]
```diff
@@ -48,6 +48,16 @@ describe('Number formatting, when Intl.NumberFormat is supported', () => {
...
+
+    describe('and given a zero valued number', () => {
+      beforeEach(() => {
+        number = 0.0;
+      });
+
+      it('should format the value', () => {
+        expect(formatNumber(number, 'en-GB')).toEqual('0');
+      });
+    });
```
Comment (pterblgh) [inline src/number/number.spec.js]: Can you please add a test scenario with zero valued numeric string as well?
